client/index.jsx

```diff
@@ -87,7 +87,7 @@ class App extends React.Component {
         </Title>
         <p />
         <EsrbPic src={esrbImage} alt={esrbCategory} />
-        {images.map((url, index) => <img src={url} alt={index + 1} />)}
+        {images.map((pic, index) => <img src={pic.url} alt={index + 1} key={pic.id} />)}
```

Comment:
This was really cool to me when eslint picked it out! Also why do you have your alt set to index + 1?

Reply:
Its just the alternate text for an image if it can't be displayed.  The index is zero based for arrays, but I wanted the numbers 1-5 to correspond with the 5 pics.  It was more of a test of assigning index than it is a real solution, I'll probably swap it out in the future to be the filename

Reply:
I see, thank you Tim!